lib/ws_servers/algos/algo_server.conf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
if we plan on having more config files like this, we should create a separate folder `configs` and store such files there. But for now, lgtm.